README.md

```diff
@@ -8,3 +8,17 @@ It covers:
   -   Add files to be tracked by Git `git add 'my-file'`.
   -   Stage changes to be commited `git add 'my-file'`.
   -   Commit changes with appropriate commit message `git commit -m 'commit message'`.
+  - Finding out information regarding what files are modified and those that are in the staging area using `git status`.
+  - Printing out all the commits that are completed using `git log`.
+  - Pushing all code from local repository to the remote repository on GitHub using `git push -u origin master`
+  -  Using `git pull origin master` to pull the latest changes from the remote repository to local
```

Comment:
This Line and the next appear to be contained in a particular step and should have it's own topic, thoughts?

Reply:
I think I should introduce a new topic that deals with working with remotes and synchronizing changes to both remote and local.

Reply:
That would work best